src/frame/headers.rs

```diff
@@ -273,6 +273,7 @@ impl fmt::Debug for Headers {
             .field("stream_id", &self.stream_id)
             .field("stream_dep", &self.stream_dep)
             .field("flags", &self.flags)
+            .field("header_block", &self.header_block)
```

Comment:
The comment right after this (while slight outdated) suggests that this field is purposefully not included :)